admin/test/test_acceptance.py

```diff
@@ -205,3 +207,19 @@ def test_non_json_message(self):
             )],
             self._convert(NON_JSON_JOURNAL_EXPORT),
         )
+
+
+class CommonOptionsParser(TestCase):
```

Comment:
The convention is that TestCase subclass names end with `Tests`....so `CommonOptionsParserTests`.